pkg/git/git.go

```diff
@@ -80,8 +80,165 @@ func (h *stiGit) Clone(source, target string) error {
 	}
 	go pipeToLog(outReader, glog.Info)
 	go pipeToLog(errReader, glog.Error)
-	return h.runner.RunWithOptions(opts, "git", "clone", "--quiet", "--recursive", source, target)
+	//previously, we were doing the git clone with --quiet, but with the piping of the wrapping cmd stdout/stderr
+	// to glog on separate threads, we were causing a deadlock processing stdout/stderr channels across 4 goroutines.
+	// a SIGQUIT produced goroutine thread dump that shows the deadlock is included  after this method for historical
+	// purposes.
+	// Since it seems contradictory to run with --quiet and also pipe git output to glog, the workaround to the apparent
+	// go issue is the remove the --quiet option.  I also saved the core file produced by the SIGQUIT.
+	// a discussion on whether this is frequent enough or bad enough to open a Go issue on github could be warranted.
+	return h.runner.RunWithOptions(opts, "git", "clone", "--recursive", source, target)
 }
+	/*
```

Comment:
we'll probably want to remove this from the final commit but worth having for discussion for now.